src/Components/WebAssembly/testassets/StandaloneApp/_Imports.razor

```diff
@@ -1,5 +1,6 @@
 @using System.Net.Http
 @using Microsoft.AspNetCore.Components.Routing
 @using Microsoft.AspNetCore.Components.Web
+@using Microsoft.AspNetCore.Components.WebAssembly.Http
```

Comment:
Hmm, this is going to make discoverability tough, given that people will want to be using this inside `.razor` files where they don't get IDE autocompletion for importing extension method namespaces.

How outrageous would it be if the extension methods went into (say) `Microsoft.AspNetCore.Components.Web`? They will still only apply to WebAssembly projects since they are in the `M.A.C.WebAssembly` package.